pkg/controller/controller_ref_manager.go

```diff
@@ -349,7 +349,12 @@ func (m *ReplicaSetControllerRefManager) AdoptReplicaSet(rs *extensions.ReplicaS
 	// Note that ValidateOwnerReferences() will reject this patch if another
 	// OwnerReference exists with controller=true.
 	addControllerPatch := fmt.Sprintf(
-		`{"metadata":{"ownerReferences":[{"apiVersion":"%s","kind":"%s","name":"%s","uid":"%s","controller":true,"blockOwnerDeletion":true}],"uid":"%s"}}`,
+		`{"metadata":{
```

Comment:
Could you fix the indention of the patch? Also it might be a good idea to add a unit test for the patch, i don't know if the patch system is picky on the indentions and newlines